.travis.yml

```diff
@@ -5,7 +5,9 @@ services:
 
 arch:
   script:
+    - "pacman -S namcap --noconfirm"
```

Comment:
I think you can install this package over the package section. 
https://github.com/mikkeloscar/arch-travis

Reply:
ya, I forgot 😅